src/workerd/util/sqlite-test.c++

```diff
@@ -334,6 +334,7 @@ void doLockTest(bool walMode) {
   KJ_EXPECT(db.run(GET_COUNT).getInt(0) == 1);
 
   // Concurrent write allowed, as long as we're not writing at the same time.
+  // NOLINTNEXTLINE(bugprone-unused-raii)
```

Comment:
What happened here?

Reply:
We're constructing a kj::Thread, and destroy it immediately since it's not assigned to a variable. This is intentional here as destroying the thread means joining it, so effectively say "run this immediately, in another thread, and synchronously wait for it to be done". I'll add a comment.